android-core/src/main/java/com/mparticle/networking/MParticleBaseClientImpl.java

```diff
@@ -133,38 +142,44 @@ protected MPUrl getUrl(Endpoint endpoint, @Nullable String identityPath, boolean
                         }
                     }
                 }
-                uri = builder.build();
+                uri = builder;
+                uri.build();
                 return MPUrl.getUrl(uri.toString(), defaultUrl);
```

Comment:
we're not doing a Uri.Builder.toString(), and before this change we were doing a Uri.toString(). was that intentional? is the underlying source code for uri/uri.builder the same? we should check the source code to confirm.